src/start/RealmScreen.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { TextInput } from 'react-native';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { styles, Screen, ErrorMsg, Button } from '../common';
+import { getAuthBackends } from '../api/apiClient';
+import config from '../config';
+import { realmAdd } from '../account/accountActions';
+import AuthBackendPicker from './AuthBackendPicker';
+
+type Props = {
```

Comment:
We should start adding `// @flow` to the beginning of files when we can. Otherwise they don't get typechecked